runtime/dancebox/src/lib.rs

```diff
@@ -967,6 +969,46 @@ impl pallet_pooled_staking::Config for Runtime {
     type WeightInfo = pallet_pooled_staking::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+    pub ParachainBondAccount: AccountId32 = AccountId32::new([42; 32]); // TODO replace by a real account
+    pub PendingRewardsAccount: AccountId32 = PalletId(*b"PENDREWD").into_account_truncating();
+    // 5%/year with 2_629_800 blocks per year -> 0.05/2_629_800 = 19/1_000_000_000
+    pub const InflationRate: Perbill = prod_or_fast!(Perbill::from_parts(19), Perbill::from_percent(1));
+
+    // 30% for parachain bond, so 70% for staking
+    pub const RewardsPortion: Perbill = Perbill::from_percent(70);
+}
+
+pub struct GetSelfChainBlockAuthor;
+impl Get<AccountId32> for GetSelfChainBlockAuthor {
+    fn get() -> AccountId32 {
+        AuthorityMapping::authority_id_mapping(Session::current_index())
+            .expect("no authorities for this session")
+            .get(&AuthorInherent::get())
+            .expect("invalid block author")
+            .to_owned()
+    }
+}
+
+pub struct OnUnbalancedInflation;
+impl frame_support::traits::OnUnbalanced<Credit<AccountId, Balances>> for OnUnbalancedInflation {
+    fn on_nonzero_unbalanced(credit: Credit<AccountId, Balances>) {
+        let _ = <Balances as Balanced<_>>::resolve(&ParachainBondAccount::get(), credit);
+    }
+}
+
+impl pallet_inflation_rewards::Config for Runtime {
+    type Currency = Balances;
+    type ContainerChains = Registrar;
```

Comment:
This is likely not right, Registrar might contain chains that are to be started next session, and those should not be rewarded. We should either:

- on initialize new session, store them in inflatoin-rewards/authority-noting pallet
- take them from the btree-map of collator-assignation